src/annotator/integrations/test/image-text-layer-test.js

```diff
@@ -277,6 +277,28 @@ describe('ImageTextLayer', () => {
     assert.calledOnce(measureImageSpy);
   });
 
+  describe('#updateSync', () => {
+    it('flushes text layer size updates immediately', () => {
+      const { image } = createPageImage();
+      const imageText = 'some text in the image';
+
+      const textLayer = createTextLayer(
+        image,
+        createCharBoxes(imageText),
+        imageText
+      );
+
+      // Spy on logic that is invoked each time a resize event is handled.
+      const measureImageSpy = sinon.spy(image, 'getBoundingClientRect');
+
+      image.style.width = '250px';
+      image.style.height = '250px';
+      textLayer.updateSync();
+
+      assert.calledOnce(measureImageSpy);
```

Comment:
Strictly speaking, isn't this testing the debounce implementation via `lodash`? In this case, I suppose it's sort of a...critical functionality of the module under test...

Reply:
This test is making sure that `updateSync()` does what it promises, regardless of how exactly it is implemented (the use of `lodash.debounce` is an implementation detail). If for example, you removed debouncing entirely or refactored it to call the function that debounce wraps directly, it should still pass.